README.md

```diff
@@ -28,7 +28,12 @@ We have used MaxText to [demonstrate high-performance, well-converging training
 Key supported features:
 * TPUs and GPUs
 * Training and Inference
-* Models: Llama2, Mistral and Gemma
+* Models: Llama 2, Llama 3, Mistral family, and Gemma, Gemma 2, Gemma 3
```

Comment:
Could you help update to 'Mistral and Mixtral family's? Thanks!

Reply:
> Could you help update to 'Mistral and Mixtral family'? Thanks!

Done